spec/std/isa/ext/Zkr.yaml

```
@@ -1,17 +1,35 @@
-# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# Copyright (c) Anirudh Narang
 # SPDX-License-Identifier: BSD-3-Clause-Clear
 
 # yaml-language-server: $schema=../../../schemas/ext_schema.json
 
 $schema: "ext_schema.json#"
 kind: extension
 name: Zkr
-long_name: Entropy Source
-description: |
-  Defines the `seed` CSR.
-  This CSR provides up to 16 physical entropy bits that can be used to seed cryptographic random bit generators.
+long_name: Entropy Source 
```

CI is failing in pre-commit due to a trailing space:

```suggestion
long_name: Entropy Source
```